requirements.txt

```diff
@@ -1,6 +1,6 @@
 flaky
 numpy
-pennylane>=0.12
+git+https://github.com/PennyLaneAI/pennylane.git
```

Comment:
Do we want to wait until after release before merging this (and then update the requirements to the latest release instead)?

Reply:
In the past we have, but we have so many plugins now, I think it is better to merge it as-is (and simply replace it with `pennylane>=0.15` once it is on PyPI).

By merging it in now, we can:

- Have its entry in the test matrix turn green,
- Keep track of it breaking (again!) between now and release :)